pdr_backend/sim/sim_engine.py

```diff
@@ -178,7 +191,16 @@ def run_one_iter(self, test_i: int, mergedohlcv_df: pl.DataFrame):
         acct_up_profit -= stake_up
         acct_down_profit -= stake_down
 
-        profit = self.sim_trader(curprice, pred_up, pred_down, conf_up, conf_down)
+        profit = self.sim_trader(
+            curprice,
+            pred_up,
+            pred_down,
+            conf_up,
+            conf_down,
+            high=high_value,
+            low=low_value,
+        )
```

Comment:
To help readability, find a way to make this take fewer lines.

For starters, you don't need `high=` and `low=`.  

Then you can get away with the following, I think:
```python
    profit = self.sim_trader(
        curprice, pred_up, pred_down, conf_up, conf_down, high_value, low_value,
    )